.github/actions/setup-java/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
     - name: Setup Java
       uses: actions/setup-java@v4
       with:
-        java-version: 21
+        java-version: 24
```

Comment:
do we need also this?

Reply:
In order to properly compile the off-heap code based on FFM, we need at least JDK 23. We should therefore now compile with JDK 25 on CI.
We should also keep the `ea` testing for `26-ea` once that is available to GH actions